src/Nav/index.js

```diff
@@ -12,32 +12,45 @@ governing permissions and limitations under the License.
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withPrefix } from 'gatsby'
-import { defaultFocus } from '../ManifestUtils'
 import { isExternal } from '../utils'
 
 import '@spectrum-css/sidenav'
 import './index.css'
 
-const nav = (data, defaultFocus, currDepth, maxDepth) => {
+const isPathSelected = (path, selected) => {
+  const compare = !selected.endsWith('/')
+    ? selected
+    : selected.substring(0, selected.length - 1)
+  return path === compare
+}
+
+const nav = (data, selected, currDepth, maxDepth) => {
   if (currDepth >= maxDepth) {
     return ''
   }
 
   return (
     <ul className='spectrum-SideNav  spectrum-SideNav--multiLevel'>
       {data.map((node, index) => {
-        const isSelected =
-          node.title === defaultFocus
-            ? 'spectrum-SideNav-item is-selected'
-            : 'spectrum-SideNav-item'
         const path = isExternal(node.path) ? node.path : withPrefix(node.path)
+        const isSelected = isPathSelected(path, selected)
+          ? 'spectrum-SideNav-item is-selected'
+          : 'spectrum-SideNav-item'
         return (
           <li className={isSelected} key={index}>
-            <a href={path} className='spectrum-SideNav-itemLink'>
+            <a
+              href={path}
+              className='spectrum-SideNav-itemLink'
+              style={{
+                paddingLeft: `calc(var(--spectrum-global-dimension-size-150) * ${
+                  currDepth + 1
+                })`
```

Comment:
It'd be good if react-spectrum provided this out of the box. Maybe we should consider opening a ticket against them?

Reply:
It's on their roadmap, SideNav. 

https://wiki.corp.adobe.com/display/RSP/React+Spectrum+v2+to+v3+component+mapping

Not that we are getting rid of the crazy stripManifestPath and defaultFocus we should consider sending PR to them to add the component to react-spectrum main.